lib/img/aws.js

```diff
@@ -45,6 +45,29 @@ AwsDriver.prototype = {
         }
       });
     });
+  },
+
+  delete: function awsDelete(key) {
+    var s3 = this._s3;
```

Comment:
Commenting for completeness: I think this is sufficiently covered by the test that mocks s3 at the network level